tests/unit/via/services/youtube_test.py

```diff
@@ -116,7 +117,6 @@ def test_get_transcript(self, db_session, svc, YouTubeTranscriptApi):
         ]
 
     @pytest.mark.usefixtures("db_session")
-    @pytest.mark.parametrize("transcript__transcript_id", ["en"])
```

Comment:
This `parametrize` was fixing the value of `transcript.transcript_id` to `"en"` because `get_transcript()`'s DB query would only find transcripts with `Transcript.transcript_id="en"`. It's no longer necessary for the test to do this.